docs/book/user-guide/component-guide/experiment-trackers/wandb.md

```diff
@@ -145,8 +145,60 @@ def tf_trainer(
         ],
     )
 
+    metric = ...
+
+    wandb.log({"<METRIC_NAME>": metric})
+```
+
+{% hint style="info" %}
+Instead of hardcoding an experiment tracker name, you can also use the 
+[Client](../../advanced-guide/environment-management/client.md) to dynamically
+use the experiment tracker of your active stack:
+
+```python
+from zenml.client import Client
+
+experiment_tracker = Client().active_stack.experiment_tracker
+
+@step(experiment_tracker=experiment_tracker.name)
+def tf_trainer(...):
     ...
 ```
+{% endhint %}
+
+### Weights & Biases UI
+
+Weights & Biases comes with a web-based UI that you can use to find further 
+details about your tracked experiments. 
+
+Every ZenML step that uses Weights & Biases should create a separate experiment 
+run which you can inspect in the Weights & Biases UI: 
```

Comment:
```suggestion
run which you can inspect on the Weights & Biases UI: 
```

Reply:
Both of the sentences are grammatically correct, but the first one, "which you can inspect in the Weights & Biases UI," is more commonly used and sounds more natural.

"In the Weights & Biases UI" indicates the location where something can be inspected, so using "in" here is appropriate. The second sentence, "which you can inspect on the Weights & Biases UI," would still be understood, but it might not be as commonly used in this context.

Reply:
Computer says no.